navigation_stage/launch/move_base_hector_5cm.launch

```diff
@@ -0,0 +1,19 @@
+<launch>
+  <master auto="start"/>
+  <param name="/use_sim_time" value="true"/>
+  <include file="$(find navigation_stage)/move_base_config/move_base.xml"/>
+  <node pkg="stage_ros" type="stageros" name="stageros" args="$(find navigation_stage)/stage_config/worlds/willow-pr2-5cm.world" respawn="false" >
+    <param name="base_watchdog_timeout" value="0.2"/>
+  </node>
+
+  <node name="hector_mapping" pkg="hector_mapping"
+        type="hector_mapping">
```

Comment:
The problem with merging this is that this introduces a dependency on `hector_mapping`, and I think this package is released pretty early in our roll out of packages in a new ROS distribution, so I'd rather not add the dependency. Is it possible that these tutorials/examples could be added to the `hector_mapping` package or to the karto packages (for the other example) instead? Or make a new package which depends on this package, and the other two?